src/components/ItemList.tsx

```diff
@@ -1,15 +1,15 @@
 import React from 'react';
+import { useSelector } from 'react-redux';
 import Item from './Item';
-import { StandartItem } from './App';
+import { StandartItem } from '../types/types';
+import { RootState } from '../store';
 
-interface Props {
-  items: StandartItem[];
-}
+function ItemList(): JSX.Element {
+  const items = useSelector<RootState, StandartItem[]>( state => state.items.items);
 
-function ItemList({ items }: Props): JSX.Element {
   return (
     <ol>
-      {items.map((item) => (
+      {items.map((item: StandartItem) => (
```

Comment:
stop typisating value which doesn't need typisation. `item`'s is implicitly set to StandartItem.